drivers/serial/uart_rtt.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2019 omSquare s.r.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/uart.h>
+#include <SEGGER_RTT.h>
+
+#if CONFIG_UART_RTT_DRIVER
+
+struct uart_rtt_config {
+	unsigned int channel;
```

Comment:
Wouldn't `u8_t` be enough for the channel number?

Reply:
Of course, but it will be still aligned unless packaged, doesn't it ?

Reply:
You can move this field to the end of the structure to avoid the padding.